packages/haul-core/src/runtime/Logger.ts

```diff
@@ -98,6 +104,18 @@ export default class Logger {
         this.inspectorClient.emitEvent(new LoggerEvent(level, args));
       }
 
+      if (this.logFile !== undefined) {
+        fs.appendFileSync(
+          this.logFile,
+          (this.logAsJson
+            ? JSON.stringify({ timestamp: new Date(), level, messages: args })
```

Comment:
why not iso here?

Reply:
`JSON.stringify` will convert it autmatically:
```
JSON.stringify(new Date())
""2019-07-17T15:57:39.215Z""
```

Reply:
TIL